src/dos/drive_overlay.cpp

```diff
@@ -302,7 +305,9 @@ bool OverlayFile::create_copy() {
 	if (drive_set != 0xff && drive_set < DOS_DRIVES && Drives[drive_set]){
 		const auto od = dynamic_cast<Overlay_Drive*>(Drives[drive_set]);
 		if (od) {
-			newhandle = od->create_file_in_overlay(GetName(),"wb+"); //todo check wb+
+			std_fs::path path = {};
+			// TODO: check wb+
+			std::tie(newhandle, path) = od->create_file_in_overlay(GetName(), "wb+");
```

Comment:
Can't rewrite to `const auto [newhandle, path] = ...`?

Reply:
Wouldn't be practical. `path` is not needed in this method (will be ignored), and `newhandle` is defined earlier (outside of this block) and it's scope cannot be reduced.